cmd/swarm/main.go

```diff
@@ -121,6 +121,10 @@ var (
 		Name:  "pss",
 		Usage: "Enable pss (message passing over swarm)",
 	}
+	DiscoveryEnabledFlag = cli.BoolFlag{
+		Name:  "discovery",
+		Usage: "Enable discovery",
+	}
```

Comment:
ok.
a definition of boolean flag here make it true by default?or it need explicitly define as true?